packages/python/plotly/plotly/figure_factory/_table.py

```diff
@@ -36,7 +36,8 @@ def create_table(
     **kwargs
 ):
     """
-    BETA function that creates data tables
+    **deprecated**, use instead the plotly.graph_objects trace 
```

Comment:
I actually realize now that this isn't an older way of making tables, but it's a convenience method for making tables, because `go.Table` is pretty hard to use. So maybe this one shouldn't be `deprecated` after all.